src/types.rs

```diff
@@ -1701,6 +2414,34 @@ impl TRS {
             .collect()
     }
     /// Do two TRSs unify?
+    ///
+    /// See [`Unification`] for more information.
```

Comment:
For conciseness and clarity, please move the link up to `unify` and then delete this sentence.